sdk/loadtestservice/Azure.ResourceManager.LoadTesting/CHANGELOG.md

```diff
@@ -1,15 +1,20 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2023-01-06)
 
 ### Features Added
 
+- Update to the latest api-version.
+
 ### Breaking Changes
 
 ### Bugs Fixed
 
 ### Other Changes
 
+- Update the Readme with samples.
+- Update test cases.
```

Comment:
Maybe we should not show the test changes here as the release will not include the test project.

Reply:
removed